public/app/features/provisioning/api/endpoints.gen.ts

```diff
@@ -752,6 +750,8 @@ export type GitHubRepositoryConfig = {
   repository?: string;
   /** Token for accessing the repository. */
   token?: string;
+  /** Workflow allowed for changes to the repository. The order is relevant for defining the precedence of the workflows. Possible values: pull-request, branch, direct-push. */
+  workflows?: string[];
```

Comment:
with the +enum annotation, this will have explicit strings

Reply:
It doesn't for me. Maybe because it's a slice and not a single value? @ryantxu can you try yourself locally?

Reply:
It doesn't for me. Maybe because it's a slice and not a single value? @ryantxu can you try yourself locally?

Reply:
Now I see the issue, the workflows has the type `('branch' | 'push')[]`, but we need it to be a standalone type, i.e., `type WorkflowOption = 'branch' | 'push', @ryantxu any idea how to generate that?